app/Models/Action.php

```diff
@@ -32,6 +35,13 @@ class Action extends Model
         'reportback', 'civic_action', 'scholarship_entry', 'anonymous', 'online', 'quiz', 'noun', 'verb', 'collect_school_id', 'volunteer_credit',
     ];
 
+    /**
+     * The relationships that should be touched on save.
+     *
+     * @var array
+     */
+    protected $touches = ['campaign'];
+
```

Comment:
This is how Algolia is able to know to pull it in yeah?

Reply:
Not Algolia but Laravel. This tells Laravel that when an Action record is updated, to let the parent Campaign model know and trigger observed events, and the Scout `toSearchableArray()`is one of them and that sends stuff to Algolia :D

Reply:
Right right, I skipped a few steps there - just meant this is what will start the chain of events, so cool!

Reply:
oh! Yep, yep! How the dance commences on the Action record side of things! 💃 🕺

Reply:
Dance: COMMENCE!!!